src/providers/ldap/sdap_id_op.c

```diff
@@ -413,18 +458,13 @@ static void sdap_id_op_hook_conn_data(struct sdap_id_op *op, struct sdap_id_conn
     op->conn_data = conn_data;
 
     if (conn_data) {
-        sdap_id_conn_data_stop_idle_timer(conn_data);
+        sdap_id_conn_data_not_idle(conn_data);
         DLIST_ADD_END(conn_data->ops, op, struct sdap_id_op*);
     }
 
     if (current && !current->ops) {
         if (current == current->conn_cache->cached_connection) {
-            int ret = sdap_id_conn_data_start_idle_timer(current);
-            if (ret != EOK) {
-                DEBUG(SSSDBG_MINOR_FAILURE,
-                      "sdap_id_conn_data_start_idle_timer() failed [%d]: %s",
-                      ret, sss_strerror(ret));
-            }
+            sdap_id_conn_data_idle(conn_data);
```

Comment:
I think `conn_data` is a typo, shouldn't it be `current`?

Reply:
Yes, good catch!  Thanks!